src/komponenter/lagreOgAvbrytKnapp/LagreOgAvbrytKnapp.tsx

```diff
@@ -52,7 +52,7 @@ const LagreOgAvbrytKnapp: FunctionComponent<Props & ButtonProps> = (props) => {
                 </div>
             )}
             <div className={cls.element('container')}>
-                <Button variant={'secondary'} onClick={props.avbryt}>
+                <Button variant={'secondary'} type="button" onClick={props.avbryt}>
```

Comment:
Hva slags funksjon har `type="button"` på en `<Button/>`?

Reply:
Jeg vet ikke hva Aksel gjør 'under the hood', men en vanlig html-`<button>` har ihvertfall `submit` som standard type. Dvs. hvis det er i en `form` så vil den sende inn formen, noe som er uheldig om det er en avbryt-knapp. 😅

Reply:
https://github.com/navikt/tiltaksgjennomforing/pull/1264/commits/d61b3794bed4605a7041d8144c070c501b8f985f